src/citrine/_rest/collection.py

```diff
@@ -1,6 +1,7 @@
 from abc import abstractmethod
 from logging import getLogger
-from typing import Optional, Union, Generic, TypeVar, Iterable, Iterator
+from typing import Optional, Union, Generic, TypeVar, Iterable, Iterator, Sequence, Dict
```

Comment:
`typing.Sequence` [is deprecated](https://docs.python.org/3/library/typing.html#typing.Sequence). Use `collections.abc.Sequence` instead.

Reply:
Yes, thanks!

Reply:
It's deprecated as of 3.9, but we support 3.7.